src/Microsoft.ML.Ensemble/Selector/SubModelSelector/BaseSubModelSelector.cs

```diff
@@ -64,7 +64,7 @@ private IEvaluator GetEvaluator(IHostEnvironment env)
                 case PredictionKind.Regression:
                     return new RegressionEvaluator(env, new RegressionEvaluator.Arguments());
                 case PredictionKind.MultiClassClassification:
```

Comment:
>MultiClassClassification [](start = 36, length = 24)

Is changing `MultiClass` to `Multiclass` a part of this PR? If not, please ignore this comment.